newsfragments/3749.documentation

```diff
@@ -0,0 +1 @@
+Fixes links to the installation and about Tahoe pages in the docs.
```

Comment:
The style guide for news fragments is https://twistedmatrix.com/trac/wiki/ReviewProcess#Newsfiles